scripts/fetch_code.ps1

```diff
@@ -73,6 +73,8 @@ $buildoutput = Join-Path $workpath "v8build\v8\out\$Platform\$Configuration"
     -replace ('V8JSIVER_V8REF', $v8Version.Replace('.', '_')) |`
     Set-Content "$SourcesPath\src\version_gen.rc"
 
+Copy-Item $SourcesPath\ReactNative.V8Jsi.Windows.nuspec $OutputPath

```

Comment:
The NuSpec filename may need to include UWP for uwp builds (since in ADO we're collating results from both win32 and uwp in one artifact and publishing both nupkgs from there).

Reply:
This has been parameterized as a NuGet variable:

```
      matrix:
        Desktop:
          NuGetId: ReactNative.V8Jsi.Windows
          NuGetExcludeExtensions: ipdb
        Universal:
          NuGetId: ReactNative.V8Jsi.Windows.UWP
          NuGetExcludeExtensions: pdb

...

buildProperties: Id=$(NuGetId)
```